build-conventions/src/main/groovy/Kherkin.sharedPublish.gradle

```diff
@@ -0,0 +1,131 @@
+plugins {
+    id 'com.android.library'
+    id 'maven-publish'
+    id 'org.jreleaser'
+}
+
+def getVersionName = { ->
+    try {
+        def stdout = new ByteArrayOutputStream()
+        exec {
+            commandLine 'git', 'describe', '--exact-match', 'HEAD'
+            standardOutput = stdout
+        }
+        return stdout.toString().trim()
+    }
+    catch (ignored) {
+        return null
+    }
+}
```

Comment:
This logic will need to change as we further figure out how we're handling the snapshot build case, but it's probably fine as-is for now if only tagged releasing is in scope.